addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -92,6 +92,7 @@ export class MassMailingHtmlField extends HtmlField {
         }
 
         this._pendingCommitChanges = (async () => {
+            this.wysiwyg.odooEditor.observerUnactive('commitChangesMassMailing');
```

Comment:
The commit message seems to be truncated or something. That being said, I don't think I understand the fix anyway. We are discarding the record so why commit changes?

Reply:
Indeed , I'm still working on this , sorry for the confusion , the other comment was old and I realized there are some other problems after